src/main/scala/latis/service/hapi/DataAlgebra.scala

```diff
@@ -21,5 +21,13 @@ trait DataAlgebra[F[_]] {
    * @param data data to write
    * @return a stream of CSV records
    */
-  def writeData(data: T): Stream[F, String]
+  def writeCsv(data: T): Stream[F, String]
```

Comment:
> Should we convert to bytes here?

We could if we wanted to be explicit, which might not be a bad idea. This would be a place to use the pipes in [`fs2.text.utf8`](https://oss.sonatype.org/service/local/repositories/releases/archive/co/fs2/fs2-core_2.13/3.1.0/fs2-core_2.13-3.1.0-javadoc.jar/!/fs2/text$$utf8$.html).